### DISCUSSION TITLE
tailwind css gap is showing even after the last element

### DISCUSSION THREAD

**Discussion by marufibnehossain** [tailwindlabs/tailwindcss#14220] [Help]
![image](https://github.com/user-attachments/assets/5f7a42ae-81e6-478e-9b51-7ef126e00e78)
I am trying to use flex and gap between two input field but Its adding gap even after the last element. this is only happening when I use gap between input field. What's the issue?

**Comment by wongjn** [comment]
Seems like there might be some elements after the last element.

**Reply by marufibnehossain** [reply]
I double checked there is only two input field inside the div.

**Reply by wongjn** [reply]
What does it look like in devtools?

**Reply by marufibnehossain** [reply]
> What does it look like in devtools?

I found the issue, its because of my password manager tool. if I disable the extension it works fine. sorry for the trouble, since the issue is not related to Tailwind.
